src/nimblepkg/packageinstaller.nim

```diff
@@ -3,7 +3,13 @@
 import os, strutils, sets, json
 
 # Local imports
-import cli, common, options, tools
+import cli, options, tools
+
+when defined(windows):
+  import version
```

Comment:
Why is this just imported for Windows?

Reply:
Because only on Windows something from this module is being used. Namely the `NimbleError` exception which is being thrown below. See [here](https://github.com/nim-lang/nimble/blob/44fc44980e7473f7c3e5c93d42aca65c07d90899/src/nimblepkg/packageinstaller.nim#L52).